src/openrct2/Game.cpp

```diff
@@ -446,7 +446,7 @@ int32_t game_do_command_p(
     {
         // Check funds
         int32_t insufficientFunds = 0;
-        if (gGameCommandNestLevel == 1 && !(flags & GAME_COMMAND_FLAG_2) && !(flags & GAME_COMMAND_FLAG_5) && cost != 0)
```

Comment:
Are you really sure that GAME_COMMAND_FLAG_2 is no longer needed?

Reply:
It was used in one function as a no money flag. I changed that function to use the no money flag instead